test/Core.Test/AutoFixture/CollectionFixtures.cs

```diff
@@ -0,0 +1,19 @@
+﻿using System;
+using AutoFixture;
+using Bit.Core.Test.AutoFixture.Attributes;
+using Bit.Core.Test.AutoFixture.OrganizationFixtures;
+
+namespace Bit.Core.Test.AutoFixture
+{
+    internal class CollectionDefaultIdAutoDataAttribute : CustomAutoDataAttribute
+    {
+        public CollectionDefaultIdAutoDataAttribute() : base(new SutProviderCustomization(), new Organization { CollectionId = default(Guid)})

```

Comment:
teeny formatting nit ('cause my eye will stick there the whole time 😉 )
```suggestion
        public CollectionDefaultIdAutoDataAttribute() : base(new SutProviderCustomization(), new Organization { CollectionId = default(Guid) })
```